server/src/libs/loom.ts

```diff
@@ -0,0 +1,85 @@
+import { Log } from "../utils/log";
+
+
+export interface LoomOEmbed {
```

Comment:
why exports? it should be only internal to this module by default unless you need it